Lesson_13/task2_json.py

```diff
@@ -0,0 +1,53 @@
+"""Validate all JSON files.
+
+Validate that all files in the ideas_for_test/work_with_json folder
+are valid json. Output the result for an invalid file through the logger
+at the error level to the file json__<your_second_name>.log.
+"""
+import json
+from pathlib import Path
+
+from logger import configure_logging
+
+_log = configure_logging(log_file='json_result.log')
+
+
+def validate_json_files(folder_path):
```

Comment:
The best way to organize atomic function for single file check. Other stuff better to put into other function of module code block ( aka json files search within certain directory )

Reply:
updated with commit e7ce4b62b49a2daa3850be70c2083336cf6a52d8